Exports com.github.sttk.linebreak.terminal

I've encountered the following error when I used this library from `cliargs-java`.

> Caused by: java.lang.IllegalAccessException: class com.sun.jna.Structure (in module com.sun.jna) cannot access class com.github.sttk.linebreak.terminal.UnixTerminal$winsize (in module com.github.sttk.linebreak) because module com.github.sttk.linebreak does not export com.github.sttk.linebreak.terminal to module com.sun.jna

However, exports it only to `com.sun.jna` with `exports ... to ...` directive.

--- a/src/main/java/module-info.java
+++ b/src/main/java/module-info.java
@@ -1,6 +1,6 @@
 /*
  * module-info class.
- * Copyright (C) 2023 Takayuki Sato. All Rights Reserved.
+ * Copyright (C) 2023-2024 Takayuki Sato. All Rights Reserved.
  */
 
 /**
@@ -8,6 +8,7 @@
  */
 module com.github.sttk.linebreak {
   exports com.github.sttk.linebreak;
-  requires com.ibm.icu;
-  requires com.sun.jna;
+  exports com.github.sttk.linebreak.terminal to com.sun.jna;
+  requires transitive com.ibm.icu;
+  requires transitive com.sun.jna;
 }

